run_docker

```diff
@@ -119,9 +119,10 @@ mkdir -p .docker_home
 
 # Start a detached container as root, add the host uname and uid to /etc/passwd,
 # then run forever
+# shellcheck disable=2086 # Intended splitting
 CONTAINER=$(\
   docker run \
-    "$PORT_CONFIGURATION_LINE" \
+    $PORT_CONFIGURATION_LINE \
```

Comment:
You can also make it explicit by using an array, then you don't need to disable the shellcheck check.

Reply:
Yes, that could be better for consistency. Thanks for the suggestion.